packages/core/src/sync-store/migrations.ts

```diff
@@ -1082,6 +1083,144 @@ AND ponder_sync."rpcRequestResults"."blockNumber" <= 9223372036854775807;
         .execute();
     },
   },
+  "2024_11_12_0_debug": {
+    async up(db) {
+      await db.schema.dropTable("callTraces").ifExists().cascade().execute();
+
+      await db
+        .deleteFrom("intervals")
+        .where("fragment_id", "like", "trace_%")
+        .execute();
+
+      await db.schema
+        .createTable("traces")
+        .addColumn("id", "text", (col) => col.notNull().primaryKey())
+        .addColumn("chainId", "integer", (col) => col.notNull())
+        .addColumn("checkpoint", "varchar(75)", (col) => col.notNull())
+        .addColumn("type", "text", (col) => col.notNull())
+        .addColumn("transactionHash", "varchar(66)", (col) => col.notNull())
+        .addColumn("blockNumber", "numeric(78, 0)", (col) => col.notNull())
+        .addColumn("blockHash", "varchar(66)", (col) => col.notNull())
+        .addColumn("from", "varchar(42)", (col) => col.notNull())
+        .addColumn("to", "varchar(42)")
+        .addColumn("gas", "numeric(78, 0)", (col) => col.notNull())
+        .addColumn("gasUsed", "numeric(78, 0)", (col) => col.notNull())
+        .addColumn("input", "text", (col) => col.notNull())
```

Comment:
Are you sure this is not null? I've definitely seen traces with an "empty" input, but maybe it's an empty string.

Reply:
https://github.com/alloy-rs/alloy/blob/5fcbb2d87100923f4274a13e3ddb54cde3bbd681/crates/rpc-types-trace/src/geth/call.rs#L24